### PR TITLE
[Travis] Switched to using Flex for testing

### PR DESCRIPTION
**Pull request by mnocon**
JIRA: https://issues.ibexa.co/browse/EZP-32201

Switched to installing our product using Flex using the new script from ibexa/ci-scripts package.

~~Contains TMP commit: https://github.com/ezsystems/ezplatform-admin-ui/pull/1667/commits/aaf205260f13351b3a62a13d1079489f49a16127~~

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -58,9 +59,7 @@
         "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
     },
     "extra": {
-        "_ezplatform_branch_for_behat_tests": "master",
```
Comment (mnocon) [inline composer.json:61]: This is no longer needed, it's set in .travis.yml now.

File: composer.json [hunk 2]
```diff
@@ -58,9 +59,7 @@
...
         "branch-alias": {
-            "dev-tmp_ci_branch": "2.3.x-dev",
```
Comment (mnocon) [inline composer.json:63]: We're using the entry for `dev-master` for setting correct branch alias in composer (https://github.com/ibexa/ci-scripts/pull/3/files#diff-caaaa590e98c1cba6c1625f4ff6625f2a5773fd955b477669356ae7655b5f69cR19)

File: .travis.yml [hunk 3]
```diff
@@ -55,18 +55,17 @@ before_install:
   - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
 install:
-  # Install packages if needed
-  - if [ "${CHECK_CS}" == "1" -o "${PHPUNIT_CONFIG}" != "" ]; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
+  - travis_retry composer install --prefer-dist --no-interaction
```
Comment (mnocon) [inline .travis.yml:58]: We need to install the ci-scripts package, so composer install is done for every job - but it should be fast as the packages will be cached on Travis.

**Review by tomaszszopinski — APPROVED** [review]
LGTM

**Review by bogusez — APPROVED** [review]

**Review by micszo — APPROVED** [review]

**Pull request merged**